src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -333,6 +333,11 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, reimbursementAcc
     const prevPolicy = usePrevious(policy);
     const prevProtectedMenuItems = usePrevious(protectedCollectPolicyMenuItems);
     const enabledItem = protectedCollectPolicyMenuItems.find((curItem) => !prevProtectedMenuItems.some((prevItem) => curItem.routeName === prevItem.routeName));
+    const [, setEnabledItemState] = useState(enabledItem);
+
+    useEffect(() => {
+        setEnabledItemState(enabledItem);
+    }, [enabledItem]);
```

Comment:
When the Accountant tab is enabled the enabled item doesn't get empty again so inside the highlight property when we're checking if enabledItem.routeName is equal to the current item routeName it will return true and cause infinite rendering, to avoid that we can re-render the WorkspaceInitialPage by using `setEnabledItemState` so the enabledItem will get empty again

Reply:
Regarding this issue, I don't think force re-render is the correct solution. If the issue is not caused by this PR, we should handle the issue separately and try to find the right root cause.

Reply:
I think this is not caused by my PR, but just need your thoughts first.
So when the accountant tab is enabled, if we go back from the Workspace clear the cache & restart, and go back to the workspace, the accountant tab is not showing until we go to another page to re-render the WorkspaceInitialPage

This is because [here](https://github.com/Expensify/App/pull/45667#discussion_r1685964313)

Reply:
@mollfpr How do you think?
Please let me know your thought

Reply:
Sorry for the delay.

I understand the issue, but it's not only happening on the accountant tab and I don't think this is the correct solution moreover just a workaround.

So, I think we should do nothing here.

Reply:
@mollfpr Ok.
I've removed the code
Thank you